subcommand/get-consul-client-ca/command_test.go

```diff
@@ -374,50 +375,6 @@ func generateCA(t *testing.T) (caPem, keyPem string) {
 	return
 }
 
-// generateServerCerts generates Consul CA
```

Comment:
Moved to `common` package since it's needed by the create-federation-secret tests.